tests/unit/backend/corpora/common/entities/test_asset.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+from backend.corpora.common.entities.asset import Asset
+from tests.unit.backend.corpora import CorporaTestCaseUsingMockAWS
+
+
+class TestDataset(unittest.TestCase):
+    def setUp(self):
+        self.uuid = "test_dataset_artifact_id"
+
+    def test__get__ok(self):
+        asset = Asset.get(self.uuid)
+        self.assertEqual(self.uuid, asset.id)
+        self.assertEqual(CorporaTestCaseUsingMockAWS.CORPORA_TEST_CONFIG["bucket_name"], asset.bucket_name)
```

Comment:
Ah, ok so what you want to do is have the TestAsset class inherit `CorporaTestCaseUsingMockAWS` ->`class TestAsset(CorporaTestCaseUsingMockAWS):`

Look at `TestDatasetValidator` for an example.

Reply:
fixed